### PR TITLE
Civilian's Delight

### PR DESCRIPTION
**Pull request by Kurfursten**
- Fully tested
- Fully sprited (includes 16 new sprites for HE lighting and one for International Drink of Mystery)
- **DOES NOT AFFECT TTVs.**

new sprites

---

Too long have we suffered under the lash of the Engineering and Science departments -- languishing as they get new features. Rise, Civilians!
- New High Efficiency bulbs (2 boxes) now spawn in the Janitor's closet (more can be ordered from cargo at 2x the price for 2/3 the boxes of normal lights). These bulbs use 1/4 the energy and can keep those high stress areas like main hallway and arrivals running much, much longer.
- Who needs science? The autolathe now produces all tier 1 components.
- The Bartender closet now contains a vial of 25u "Black Food Coloring". When placed in any drink container, ANY amount of this reagent will cause it to be impossible to examine for contents. Placing it in a drinking glass causes it to become an International Drink of Mystery. When transferred to a non-drink container (e.g.: bottle or beaker), it reveals its true nature. Still possible to scan with a reagent scanner.
- NT scientists have reduced the weight and broken access of coinboxes, they can now be properly held in one hand and opened.
- In a stunning double-breakthrough, NT researchers have developed a new "used recharge pack". Produced when you insert 4 sheets of cardboard to a vendor before disassembling, the used recharge pack contains all the remaining contents of the vendomat! Now you don't have to waste everything inside if you want to scan the circuitboard or switch out the vendor type.
- Shaft Miners and Cargo Technicians no longer have QM Office access.
- Nitrogen tank moved to the bottom of O2 lockers so that Oxygen tanks are easier to grab. Gas the Vox: race war now.

### CODE REVIEW

**Comment by Exxion** [comment]
Now just to port /tg/'s SUPER ADVANCED JANITECH
Or at least the holographic sign projector
I guess the existence of the upgraded janicart makes the rest a bit unnecessary

**Comment by ComicIronic** [comment]
As I pointed out on the other PR before it got deleted, the nature of the coinbox is heavily intended. It is the largest storage item in the game that doesn't require research. The point of it needing two hands is that it's meant to be inconvenient to carry around, so that people don't just use them willy-nilly.

**Comment by Kurfursten** [comment]
@ComicIronic As far as I know, it's actually impossible to open the coinbox due to its nature as a musttwohand item. Do you know of a way to actually get items out of it as it is in its current state?

**Comment by ComicIronic** [comment]
Drag it on to the mob sprite while it's on the floor.

**Comment by NigglyWiggly** [comment]
ss 2015-05-01 at 06 27 30

**Comment by Kurfursten** [comment]
@ComicIronic Wouldn't the proper way of doing what you want be to restrict the coinbox to coin items? e.g.: the pill collector can hold a very large number of items and does not require research

**Comment by NigglyWiggly** [comment]
ss 2015-05-01 at 06 31 53

This might be just a tiny bit exploitable

**Comment by DeityLink** [comment]
>adding ways to save up energy
>every single engine on the station can singlehandedly power the station 3 to 100 times.

yeah no, I'd rather had lights that consume ten times more power but slowly change color over time.

**Comment by Kurfursten** [comment]
@DeityLink Not in the scope of this PR; pomf has already told me this is too non-atomic; can't expand

However, this DOES lay the framework for easily moddable power consumption if you want to build on top of that with your own colored light PR.

**Comment by DeityLink** [comment]
that's cool then

**Comment by Kurfursten** [comment]
Here is the new implementation I have coded for the coinbox:
* Items held size dropped to 1
* Can now only hold coins and vouchers (both needed by the code)
* The size of the box itself reduced to 2

There's no reason for this coin-holding object to be xboxheug. This will go up with the next commit: please find any other issues before I commit it.

File: code/game/objects/items/weapons/storage/lockbox.dm [hunk 1]
```diff
@@ -197,12 +197,12 @@
 	name = "coinbox"
 	desc = "A secure container for the profits of a vending machine."
 	icon_state = "coinbox+l"
-	w_class = 5
-	max_w_class = 4
+	w_class = 2
+	max_w_class = 1
+	can_hold = list("/obj/item/voucher","/obj/item/weapon/coin")
 	max_combined_w_class = 30
 	force = 15
```
Comment (ComicIronic) [inline code/game/objects/items/weapons/storage/lockbox.dm]: Change this - it no longer has a reason to be so hefty.

File: code/game/objects/items/weapons/storage/lockbox.dm [hunk 2]
```diff
@@ -210,3 +210,4 @@
 	icon_locked = "coinbox+l"
 	icon_closed = "coinbox"
 	icon_broken = "coinbox+b"
+	can_hold = list("/obj/item/weapon/reagent_containers/food/snacks/donut")
```
Comment (ComicIronic) [inline code/game/objects/items/weapons/storage/lockbox.dm]: Why would you double-define

**Comment by Kurfursten** [comment]
@ComicIronic Recommend NFD

**Comment by FudgePucker** [comment]
@DeityLink Oh man colored lights like goon has would be awesome.
I want to replace the lights in botany with black lights and smoke godly blunts while admiring my poster collection.

**Pull request merged**